src/main/java/zeobase/zbtechnical/challenges/utils/CustomStringUtils.java

```diff
@@ -0,0 +1,22 @@
+package zeobase.zbtechnical.challenges.utils;
+
+import org.springframework.lang.Nullable;
+import org.springframework.util.StringUtils;
+
+public class CustomStringUtils extends StringUtils {
+
+    public static boolean validateEmptyAndWhitespace(@Nullable String str) {
+
+        if(!hasLength(str)) {
```

Comment:
이미 StringUtils에 거의 동일한 기능이 있네요.
아래와 같이 쓰면 코드도 줄이고 성능도 비슷할듯해요.
```suggestion
        return StringUtils.hasLength(str) && !StringUtils.containsWhitespace(str)
```